zcutil/build.sh

```diff
@@ -88,11 +88,11 @@ then
     shift
 fi
 
-# If --disable-proton is the next argument, disable building Proton code:
-PROTON_ARG=''
-if [ "x${1:-}" = 'x--disable-proton' ]
+# If --enable-proton is the next argument, enable building Proton code:
+PROTON_ARG='--enable-proton=no'
+if [ "x${1:-}" = 'x--enable-proton' ]
 then
-    PROTON_ARG='--enable-proton=no'
+    PROTON_ARG=''
```

Comment:
Nitpick: I suspect many users do `./configure …; make; make install`, and I'd like to change the default for them. However, I'm content to rely on the underspecified policy that "`build.sh` is the only officially supported build path for Zcash" (despite my desire for #58 eventually).